CIP-calidus-key/README.md

```diff
@@ -0,0 +1,373 @@
+| 0  | Ed25519 Key Signature  | `[0]`                               | The most basic and simplistic approach to signing and validation. In this case the Registration Witness object could contain one or more pubkey + signed witness objects. The payload to be signed should be the hex-encoded CBOR representation of the Registration Payload object.                                                                                                                                                       |
+| 1  | Beacon/Reference Token | `[1, [h'<policyId>',h'<assetId>']]` | Similar to the approach utilized by [CIP-27](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027). We could attach this metadata during a mint transaction for a specially formatted token under the policy ID in question. CIP-27 uses a "nameless" token that has an empty "Asset ID" for example. This may be a validation method that lends itself better to supporting token projects that are minted via Smart Contract. |
+| 2  | CIP-0008 Signature     | `[2]`                               | Follow the specifications in [CIP-0008] to verify signatures                                                                                                                                                                                                                                                                                                                                                                               |
+
+**Examples:**
+
+`[0]`<br />
+`[1, [h'<policyId>',h'<assetId>']]`<br />
+`[2]`
+
+#### 4. Nonce
+
+The nonce value is utilized to prevent a replay attack vector. The nonce value
+should be an unsigned integer value that is always at least one greater than the
+previously registered value. It is recommended that you use the current absolute
+slot height of the chain when constructing a new registration to ensure that
+this value always increases.
+
+**Example:**
+
+`12345`
+
+> Note: The following map keys and values are optional
+
+#### 5. Data Oracle URI
+
+To be utilized and expanded upon in a separate CIP, this should be a valid URI
+pointing to a source of additional, potentially dynamic information relating to
+the project and/or the tokens minted under it.
+
+> This is not currently used for stake pool operator registrations.
+
+#### 6. CIP-Specific Information
+
+This entry, if present, should be a CIP ID indexed object containing additional
+information pertaining to that CIP. When and where possible the CIP-Specific
+registration should follow the CBOR-like declaration syntax to ensure that the
+content is well-formed and easily parseable.
+
+> There are currently no Stake Pool-specific CIPs to be used here but could be
+> added in the future.
+
+#### 7. Calidus Key
```

Comment:
might be worth considering defining a derivation path for these keys

Reply:
hey @Ryun1, yes this is something we can consider in an upgrade path. as those keys are hot/warm ones, they can be renewed fairly easy. 

we did not went with a fixed derivation path for now, because we would than also need support on the hw-wallet side for that too. and as financing/budget is currently in limbo mode, we did not want to generate extra costs or delays for now.

the suggestion currently is to use the standard payment path, and we have the tools on the cli that would also output mnemonics to reimport in a lightwallet for browser usage.